contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -2539,6 +2539,18 @@ $$
 LANGUAGE 'pltsql';
 GRANT EXECUTE ON PROCEDURE sys.sp_helpdbfixedrole TO PUBLIC;
 
+
+CREATE OR REPLACE PROCEDURE sys.sp_set_session_context ("@key" sys.sysname, 
+	"@value" sys.SQL_VARIANT, "@read_only" sys.bit = 0)
+AS 'babelfishpg_tsql', 'sp_set_session_context'
+LANGUAGE C;
+GRANT EXECUTE ON PROCEDURE sys.sp_set_session_context TO PUBLIC;
```

Comment:
Is execute the correct permissions level?

Reply:
Yes